Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/RouteExtra.hs

```diff
@@ -21,25 +21,6 @@ import qualified Storage.Beam.Route as BeamR
 import Storage.Queries.OrphanInstances.Route
 
 findAllMatchingRoutes :: (MonadFlow m, EsqDBFlow m r, CacheFlow m r) => Maybe Text -> Maybe Integer -> Maybe Integer -> Id MerchantOperatingCity -> VehicleCategory -> m [Route]
```

Comment:
Why deleted ?

Reply:
Sorry, initially both parts looked same, I thought I might have copied it, thus deleted. I forgot to check it again. I will add that part back.